notifications/detekt.yml

```diff
@@ -26,3 +26,9 @@ style:
   ReturnCount:
     active: true
     max: 10
+
+complexity:
+  TooManyFunctions:
+    excludes: ["**/util/**", "**/test/**"]
+  ComplexMethod:
+    threshold: 20
```

Comment:
I think it is better not to add generic exceptions to rules. we can individually suppress method if required